package.json

```diff
@@ -33,6 +33,7 @@
     "report-coverage": "cat coverage/lcov.info | coveralls",
     "test": "jest"
   },
+  "packageManager": "pnpm@7.17.0",
```

Comment:
Seems like this is this for [corepack](https://nodejs.org/api/corepack.html)?

Reply:
Yeah, [`pnpm/action-setup`](https://github.com/pnpm/action-setup) requires a version to be set either directly via the `version` option (see https://github.com/open-cli-tools/concurrently/pull/354#discussion_r950709085) or via the `packageManager` property in `package.json`.

If you're okay with this, I'm going to revert it and use the `version` option of the action instead, which allows us to specify a range (e.g. `7`) instead of a fixed version (like we have now, `7.17.0`).

Reply:
> allows us to specify a range (e.g. 7) instead of a fixed version

SGTM

Reply:
Nice, thanks! I'll update the PR in a few minutes 👍